src/api/app/models/repository.rb

```diff
@@ -2,6 +2,7 @@ class Repository < ApplicationRecord
   belongs_to :project, foreign_key: :db_project_id, inverse_of: :repositories
 
   before_destroy :cleanup_before_destroy
+  before_save :set_remote_project_name_to_empty_string
```

Comment:
I guess the other option would be to validate that its not nil? Maybe thats better?